10up/Ad-Refresh-Control
# Add PHP version check for plugin.

### Description of the Change

Add minimum version check for the plugin before loading it. This will ensure plugin update doesn't break the site that don't match our minimum PHP version.

<img width="1076" alt="image" src="https://github.com/10up/Ad-Refresh-Control/assets/10358350/42e20062-d042-457e-85a1-15b3238b27d5">



Closes #122

### How to test the Change


1. Setup a WordPress environment running PHP 7.3
2. Install Ad-Refresh-Control  plugin by downloading it with this [feature](https://github.com/10up/Ad-Refresh-Control/archive/refs/heads/fix/122-php-ver-check.zip) branch.
3. Verify that plugin functionality doesn't load and you see an admin error message for php version. ( refer above screenshot in Description )

### Changelog Entry


> Fixed - Better error handling for environments that don't match our minimum PHP version

### Credits

Props @dkotter, @rahulsprajapati 


### Checklist:
